cmd/promtool/main.go

```diff
@@ -57,6 +59,7 @@ func main() {
 	checkMetricsCmd := checkCmd.Command("metrics", checkMetricsUsage)
 
 	queryCmd := app.Command("query", "Run query against a Prometheus server.")
+	queryCmdFmt := queryCmd.Flag("format", "Output format of the query.").Short('o').Default("promql").Enum("promql", "json")
```

Comment:
Not sure why the help text doesn't tell that valid values for this flag are `promql` or `json`.

Reply:
:/